applications/OptimizationApplication/python_scripts/responses/response_routine.py

```diff
@@ -0,0 +1,137 @@
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.OptimizationApplication as KratosOA
+from KratosMultiphysics.OptimizationApplication.controls.control import Control
+from KratosMultiphysics.OptimizationApplication.responses.response_function import ResponseFunction
+from KratosMultiphysics.OptimizationApplication.controls.master_control import MasterControl
+from KratosMultiphysics.OptimizationApplication.utilities.union_utilities import SupportedSensitivityFieldVariableTypes
+
+class ResponseRoutine:
```

Comment:
```suggestion
class ControlledResponse:
```

Reply:
After thinking about the name, I like ResponseRoutines more. We can add a description for a class:

"""A class ResponseRoutine" adds optimization-specific utilities to simplify routines and synchronization between the control field from algorithms and analysis models."""